content/blog/meet-our-new-hpe-dev-team-representative-at-hpe-technology-and-solution-summit-2023.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Meet our new HPE DEV Team Representative  at HPE Technology and Solution
+  Summit 2023!
+date: 2023-03-30T13:29:10.157Z
+author: Frederic Passeron
+authorimage: /img/frederic-passeron-hpedev-192.jpg
+disable: false
+tags:
+  - HPE TSS
+  - HPE DEV
+---
+![](/img/tss2023.png "HPE TECHNOLOGY AND SOLUTIONS SUMMIT 2023")
+
+HPE TECHNOLOGY AND SOLUTIONS SUMMIT 2023 (HPE TSS 2023) goes physical again this year in sunny Barcelona. After several virtual editions, you will have the pleasure to engage with HPE Developer Community representative. All our sessions will be hold by our new team member : Mathieu Losmede. 
+
+H﻿PE TSS is the largest HPE technical event taking place in EMEA. Whether you are an HPE Presales or Partner, you will have the opportunity to attend technical breakout, Hands-on Labs, as well as strategic keynotes, and also finally get to meet your peers in person!
+
+T﻿he final decision to turn the event into a physical one came late this year. As a result, the HPE DEV team will not be able to bring the full physical Hack Shack back in HPE TSS 2023. However, we still come with a few sessions, some Hands-on labs, and a few surprises...
+
+**Meet our new team member: Mathieu Losmede**
+
+M﻿athieu joined our team last summer. He came in with a strong HPE GreenLake background. He was part of the xxxx team where he worked on..... . As the team focus lately shifted to HPE GreenLake Cloud Platform, his arrival  was very fortunate. Mathieu will be all over the place in Barcelona. One might wonder whether he managed to discover the secret of ubiquity...He will delivering several technical breakouts either on his own or in partnership with some other Business Unit representatives (HPE GreenLake, Storage, Compute, and Aruba). He will also be co presenting in other BUs ' sessions. On top of these, he will deliver as well three Hands-on labs based on our well-known [Workshops-on-Demand](https://developer.hpe.com/hackshack/workshops) project.
+
+![](/img/tss2023-wod.png "Workshops-on-Demand")
+
+Look for the HPE DEV track in the HPE TSS2023 agenda to get details about all our sessions.
+
+**A﻿nd techies, they wanna have fun!**
+
+A﻿s always, the HPE DEV team brings in new ways to learn in a fun way. This year at TSS2023 in Barcelona, we propose you to participate to a treasure hunt. Do not worry, no long wlaking needed there, we won't be sending you looking for clues in the city of Barcelona. All you will be needing  is an internet browser and a few minutes. The treasure hunt will be available from the HPE TSS mobile app. It will start on Monday the 22nd of May at 12.00 pm CEST and end on Thursday the 25th of May at 10.00 CEST.  
+
+The 3 winners will be offered their prizes on Thursday the 25th of may on the show floor at the HPE GreenLake booth at 12:15pm CEST. Terms and Conditions to participate to the treasure hunt are available here.
```

Comment:
Recommended rephrasing:
"Three winners will be selected. The winners will receive their prizes on Thursday, May 25th on the show floor at the HPE GreenLake booth at 12:15pm CEST. Please read the Terms and Conditions for participation in the HPE Developer Community Treasure Hunt found here."